### PR TITLE
docs(pt): replace 'vua' with 'vue' in multiple files

### PR DESCRIPTION
**Pull request by viniciusdeliz**
## Description of Problem
Several "Vua" instances found while reading the documentation

## Proposed Solution
Replace "Vua" with "Vue"

### CODE REVIEW

File: src/guide/reusability/composables.md [hunk 1]
```diff
@@ -86,7 +86,7 @@ const { x, y } = useMouse()
   Mouse position is at: {{ x }}, {{ y }}
 </div>
 
-[Experimente-o na Zona de Testes](https://play.vuejs.org/#eNqNkj1rwzAQhv/KocUOGKVzSAIdurVjoQUvJj4XlfgkJNmxMfrvPcmJkkKHLrbu69H7SlrEszFyHFDsxN6drDIeHPrBHGtSvdHWwwKDwzfNHwjQWd1DIbd9jOW3K2qq6aTJxb6pgpl7Dnmg3NS0365YBnLgsTfnxiNHACvUaKe80gTKQeN3sDAIQqjignEhIvKYqMRta1acFVrsKtDEQPLYxuU7cV8Msmg2mdTilIa6gU5p27tYWKKq1c3ENphaPrGFW25+yMXsHWFaFlfiiOSvFIBJjs15QJ5JeWmaL/xYS/Mfpc9YYrPxl52ULOpwhIuiVl9k07Yvsf9VOY+EtizSWfR6xKK6itgkvQ/+fyNs6v4XJXIsPwVL+WprCiL8AEUxw5s=)
+[Experimente-o na Zona de Testes](https://play.vuejs.org/#eNqNkj1rwzAQhv/KocUOGKVzSAIdurVjoQUvJj4XlfgkJNmxMfrvPcmJkkKHLrbu69H7SlrEszFyHFDsxN6drDIeHPrBHGtSvdHWwwKDwzfNHwjQWd1DIbd9jOW3K2qq6aTJxb6pgpl7Dnmg3NS0365YBnLgsTfnxiNHACVueKe80gTKQeN3sDAIQqjignEhIvKYqMRta1acFVrsKtDEQPLYxuU7cV8Msmg2mdTilIa6gU5p27tYWKKq1c3ENphaPrGFW25+yMXsHWFaFlfiiOSvFIBJjs15QJ5JeWmaL/xYS/Mfpc9YYrPxl52ULOpwhIuiVl9k07Yvsf9VOY+EtizSWfR6xKK6itgkvQ/+fyNs6v4XJXIsPwVL+WprCiL8AEUxw5s=)
```
Comment (mumumilk) [inline src/guide/reusability/composables.md:89]: just a heads up that this link appears to be broken now

**Comment by mumumilk** [comment]
do you think we should also change instances of `a Vue`, `da Vue` to `o Vue`, `do Vue` ?

**Comment by nazarepiedady** [comment]
> do you think we should also change instances of `a Vue`, `da Vue` to `o Vue`, `do Vue` ?

@mumumilk, the documentation will still be as it is, because it is extremely necessary to observe the grammatical rules of Portuguese. **Vue** is not a language but an abstraction of a language so it does not have the right to use the article **O**, and in Portuguese, we use a lot of subtle words, so when we say **A vue** there is a word between these two words that is **abstração**, so to not repeatedly write **A abstração Vue** the documentation use just **A vue**, the same rule is applied to **da Vue** where would be written **da abstração vue** but it just use **da vue**

**Comment by viniciusdeliz** [comment]
> > do you think we should also change instances of `a Vue`, `da Vue` to `o Vue`, `do Vue` ?
> 
> @mumumilk, the documentation will still be as it is, because it is extremely necessary to observe the grammatical rules of Portuguese. **Vue** is not a language but an abstraction of a language so it does not have the right to use the article **O**, and in Portuguese, we use a lot of subtle words, so when we say **A vue** there is a word between these two words that is **abstração**, so to not repeatedly write **A abstração Vue** the documentation use just **A vue**, the same rule is applied to **da Vue** where would be written **da abstração vue** but it just use **da vue**

"Abstração" however is a word that you defined by yourself to be the direct translation of "framework". Framework could be translated to other words, "arcabouço" as an example, which would then use the "O" article.

**Comment by nazarepiedady** [comment]
@mumumilk the documentation use the **abstração** because it follows a debate about the usage of the word **framework** where many experienced programmers defend and prefer to use the word **abstraction** in the place of **framework**. Normally I do some instigations about the terms and alternatives before deciding to use some translated word, but as I said the documentation is using the **a** not just because **framework** was translated to **abstração** but because *Vue **abstract** the JavaScript*.

Those choices of words are made for translations done exclusively for Brazillian Portuguese. This is not the case. We always use dictionaries to read any content even if this content is written in Portuguese, and for us, the majority of the Portuguese content available on the internet is in Brazillian Portuguese, even for this case we just use dictionaries. It is too simple, just use the dictionary as we do.

I do not intend to answer any questions about this point. I will link an issue where people interested can find some of my points about this subject [vuejs-br/docs-next/issue/292](https://github.com/vuejs-br/docs-next/issues/292).

**Pull request merged**